### ISSUE TITLE
Notifications don't form a group

### ISSUE THREAD

**Issue by Terrance** [ImranR98/Obtainium#1999]
#### Describe the bug

"Error Checking for Updates" notifications are generated one per host, but each appear separately.  This is a minor nuisance on spotty connections when all the update checks fail, as they must be dismissed individually.

Most other applications have their notifications grouped, which allows them to be snoozed or dismissed as a group.

#### Screenshot

<img src="https://github.com/user-attachments/assets/86d186a4-4d42-405d-951c-501aaec9e7b6" width="300">

Note how the Termux notifications below are grouped, and the group name can be swiped to act on all the notifications at once.

#### Device information

 - Device: OnePlus Nord CE
 - OS: Android 13
 - Obtainium Version: 1.1.32

**Comment by ImranR98** [comment]
Leaving this here for my reference later: https://pub.dev/packages/flutter_local_notifications#grouping-notifications

**Comment by ImranR98** [comment]
Blocked by https://github.com/MaikuB/flutter_local_notifications/issues/2486

**Comment by Terrance** [comment]
As the upstream issue was closed, is this still blocked?

**Comment by ImranR98** [comment]
Unblocked. Looks like I haven't understood the different options.

**Comment by master-bob** [comment]
This is not present on Android 16 OneUI 8 and Obtainium 1.6.10

**Comment by Terrance** [comment]
For completeness, still an issue here on Obtainium 1.6.11, so may be device-specific or repro'd differently above?

<img width="300" alt="Ungrouped notifications" src="https://github.com/user-attachments/assets/ce908309-29b6-426c-b193-889ddb790aee" />

**Comment by master-bob** [comment]
TL;DR: Extra details (verification of previous claim of grouping). Android 16, OneUI 8.0, Obtanium 1.6.11

Sorry, I misread the details of this ticket. I thought it was about notifications in general + errors. If I recall correctly I have seen an error notification grouped with a normal successful notification. I added a second application which will generate an error checking for updates, set the background update interval to 15 minutes, and then waited,

Here are the results:

<img width="539" height="245" alt="Image" src="https://github.com/user-attachments/assets/1bd0eab8-8f16-4794-b32d-551cec8a99ed" />

<img width="540" height="569" alt="Image" src="https://github.com/user-attachments/assets/eea3d0a8-7007-4e26-82a3-4255f26039c5" />

It may be an Android 13 defect / bug.

Edit: Further details. I saw the ungrouped error notifications, two of them before I left the notification pull down. When back on the home screen, I went to take a screenshot, but the notifications where then grouped. They two ungrouped appeared after changing background updates back to 1 day; it is possible they where ungrouped as they appeared when the notification pull down was open, bit this is not clear.
